dapps/react-dapp-v2/src/contexts/JsonRpcContext.tsx

```diff
@@ -315,10 +317,14 @@ export function JsonRpcContextProvider({
           throw new Error(`Missing chain data for chainId: ${chainId}`);
         }
 
-        const valid = _verifyEip155MessageSignature(
-          message,
+        const hashMsg = hashPersonalMessage(message);
+        const rpc =
+          rpcProvidersByChainId[Number(chainId.toString().split(":")[1])];
```

Comment:
Should we throw an error if there's no rpcProvider for the given chainId ?

Reply:
Good call thank you. Actually helped me clear up the existing check there which was redundant 👍